### PR TITLE
Bug 1761043: provides a temporal fix to improve CRD publishing e2e tests in an HA setup

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: vendor/k8s.io/kubernetes/test/e2e/apimachinery/crd_publish_openapi.go [hunk 1]
```diff
@@ -481,7 +481,7 @@ func setupCRD(f *framework.Framework, schema []byte, groupSuffix string, version
 
 func setupCRDAndVerifySchema(f *framework.Framework, schema, expect []byte, groupSuffix string, versions ...string) (tCRD *crd.TestCrd, err error) {
 	defer func() {
-		if err != nil {
+		if err == nil {
```
Comment (p0lyn0mial) [inline vendor/k8s.io/kubernetes/test/e2e/apimachinery/crd_publish_openapi.go:484]: we want to sleep only if there were no errors :)

**Comment by soltysh** [comment]
/retitle Bug 1761043: provides a temporal fix to improve CRD publishing e2e tests in an HA setup

**Pull request merged**